Makefile

```
@@ -277,25 +272,36 @@ publish-core:
 publish-helpers:
 	bin/release_npm.sh @skipruntime/helpers skipruntime-ts/helpers/package.json $(OPT)
 
-.PHONY: publish-tests
-publish-tests:
-	bin/release_npm.sh @skipruntime/tests skipruntime-ts/tests/package.json $(OPT)
-
 .PHONY: publish-wasm
 publish-wasm:
 	bin/release_npm.sh @skipruntime/wasm skipruntime-ts/wasm/package.json $(OPT)
 
-.PHONY: publish-addon
-publish-addon:
-	bin/release_npm.sh @skipruntime/addon skipruntime-ts/addon/package.json $(OPT)
+.PHONY: publish-native
+publish-native:
+	bin/release_npm.sh @skipruntime/native skipruntime-ts/addon/package.json $(OPT)
 
 .PHONY: publish-server
 publish-server:
 	bin/release_npm.sh @skipruntime/server skipruntime-ts/server/package.json $(OPT)
 
-.PHONY: publish-runtime
-publish-runtime:
-	bin/release_npm.sh @skipruntime/runtime skipruntime-ts/runtime/package.json $(OPT)
-
 .PHONY: publish-all
-publish-all: clean publish-std publish-json publish-date publish-api publish-core publish-helpers publish-tests publish-wasm publish-addon publish-runtime publish-server
+publish-all: clean publish-std publish-json publish-date publish-core publish-helpers publish-wasm publish-native publish-server
+
+.PHONY: skargo
+skargo: $(eval SKARGO:=$(shell which skargo))
+ifeq ($(SKARGO),)
+	./bin/check-config.sh
+	git submodule update --init --recursive
+	cd skiplang/compiler && make STAGE=0
+else
+	@echo "SKARGO installed."
+endif
+
+.PHONY: skipruntime-native
+skipruntime-native: skargo
+ifeq ($(SKARGO),)
+	PATH=$(realpath ./skiplang/compiler/stage0/bin):$(PATH) skargo b -r --manifest-path=skipruntime-ts/native/Skargo.toml --lib --out-dir=$(HOME)/.local/lib
+else
+	skargo b -r --manifest-path=skipruntime-ts/native/Skargo.toml --lib --out-dir=$(HOME)/.local/lib
+endif
```

Implicitly installing the skiplang tool chain to `~/.local/` is quite intrusive.
```suggestion
.PHONY: skipruntime-native
skipruntime-native:
ifeq ($(SKARGO),)
	which skargo > /dev/null || (echo "Skiplang tool chain not found. Make sure it is available in \$PATH"; false)
	skargo b -r --manifest-path=skipruntime-ts/native/Skargo.toml --lib --out-dir=$(HOME)/.local/lib
```